azure-functions/create-function-app-connect-to-document-db/create-function-app-connect-to-documentdb.sh

```diff
@@ -0,0 +1,25 @@
+# login to azure 
+az login
+
+# create a resource group with location
+az group create --name myResourceGroup --location westeurope
+
+# create a storage account 
+az storage account create --name myfunctionappstorage --location westeurope --resource-group myResourceGroup --sku Standard_LRS
+
+# create a new function app, assign it to the resource group you have just created
+az functionapp create --name myFunction --resource-group myResourceGroup --storage-account myfunctionappstorage --consumption-plan-location westeurope
+
+# create documentdb database. 
+az documentdb create --name myDocumentDB --resource-group myResourceGroup
+
+# clean up
```

Comment:
`az group delete` is captured in the wrapper and doesn't need to be captured here.